panpipes/funcs/io.py

```diff
@@ -329,9 +330,10 @@ def gen_load_spatial_jobs(caf, mode_dictionary = {}):
         else:
             outfile = outfile + ".h5mu"
         sample_id = caf['sample_id'][nn]
-        yield sample_id, outfile, \
-              spatial_path, spatial_filetype, spatial_counts, spatial_metadata, spatial_transformation
-# ###############################
+        yield spatial_path,  outfile, \
```

Comment:
The first two arguments of @files have to be the input file and output file so I have switched spatial_path and sample id), doesn't matter about the rest.